src/main/java/org/store/book/User.java

```diff
@@ -0,0 +1,95 @@
+package org.store.book;
+
+public class User {
```

Comment:
Remove unneccessary setter methods for members that would not change once user is initialized.